modules/data-streams/src/javaRestTest/java/org/elasticsearch/datastreams/logsdb/qa/StandardVersusLogsIndexModeRandomDataChallengeRestIT.java

```diff
@@ -126,10 +126,6 @@ protected XContentBuilder generateDocument(final Instant timestamp) throws IOExc
         var document = XContentFactory.jsonBuilder();
         dataGenerator.generateDocument(document, doc -> {
             doc.field("@timestamp", DateFormatter.forPattern(FormatNames.STRICT_DATE_OPTIONAL_TIME.getName()).format(timestamp));
```

Comment:
Why this is not included in `withPredefinedFields` as well?

Reply:
Great question!

Reply:
Ah, this is because there is already a mapping for this in default logsdb template and i wanted to use that.